testng.xml

```diff
@@ -1,9 +1,37 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
-<suite name="Loan Department">
-  <test thread-count="5" name="Personal Loan0">
-    <classes>
-      <class name="test.Basics"/>
-    </classes>
+<suite name="Suite">
+
```

Comment:
delete whitespace

Reply:
done. Deleted more whitespaces in other files